orttraining/orttraining/test/python/onnxruntime_test_postprocess.py

```diff
@@ -209,6 +209,70 @@ def _bert_helper(self, onnx_model):
         assert model_info[0].name == expand_nodes[0].output[0]
         assert model_info[0].type == onnx_model.graph.input[0].type
 
+    def test_extra_postpass(self):
+        def postpass_replace_first_add_with_sub(model):
```

Comment:
Should we test both cases where the model is direct ONNX model and when it is a torch.nn.Module being converted into ONNX?